exe/kubernetes-render

```diff
@@ -22,11 +22,11 @@ ARGV.options do |opts|
 end
 
 templates = ARGV
-logger = KubernetesDeploy::FormattedLogger.build(verbose_prefix: false)
+logger = Krane::FormattedLogger.build(verbose_prefix: false)
 
 begin
-  KubernetesDeploy::OptionsHelper.with_processed_template_paths(template_dir) do |dir|
-    runner = KubernetesDeploy::RenderTask.new(
+  Krane::OptionsHelper.with_processed_template_paths(template_dir) do |dir|
+    runner = Krane::RenderTask.new(
```

Comment:
Since this is the kubernetes-deploy executable, we should probably use the `KubernetesDeploy` ruby API here (same with all the old exes). If we don't, we definitely need to do two separate tophats, one of the exe and one in a console.

Reply:
Done! I will 🎩 the exe's now - thanks for your help!